libraries/rush-lib/src/cli/RushXCommandLine.ts

```diff
@@ -45,129 +47,139 @@ interface IRushXCommandLineArguments {
 }
 
 export class RushXCommandLine {
-  public static launchRushX(launcherVersion: string, isManaged: boolean): void {
-    RushXCommandLine._launchRushXInternal(launcherVersion, { isManaged });
+  public static launchRushX(launcherVersion: string, options: ILaunchRushXInternalOptions): void {
+    try {
+      const args: IRushXCommandLineArguments = this._getCommandLineArguments();
+      const rushConfiguration: RushConfiguration | undefined = RushConfiguration.tryLoadFromDefaultLocation({
+        showVerbose: false
+      });
+      const eventHooksManager: EventHooksManager | undefined = rushConfiguration
+        ? new EventHooksManager(rushConfiguration)
+        : undefined;
+
+      const ignoreHooks = process.env.INSIDERUSH === '1';
+      eventHooksManager?.handle(Event.preRushx, false /* isDebug */, ignoreHooks);
```

Comment:
Does `rushx` not currently support the `--debug` flag?

Reply:
Apparently it does not:
```
usage: rushx [-h]
       rushx [-q/--quiet] <command> ...

Optional arguments:
  -h, --help            Show this help message and exit.
  -q, --quiet           Hide rushx startup information.
  ```
  
  However the design is that `rushx` parameters must precede the `<command>` (whose name cannot start with `-`).  So, it would be easy to add support for `--debug` if needed.

Reply:
Added --debug parameter